### DISCUSSION TITLE
Como solicitar um PR quando o teste falha (e nao tem a ver com a alteracao)

### DISCUSSION THREAD

**Discussion by eternauta123** [po-ui/po-angular#1383] [Angular]
Somente fiz alteracao de traducao, mas o test ui nao esta passando com erro de injection. https://github.com/po-ui/po-angular/runs/8041269687?check_suite_focus=true

**Comment by alinelariguet** [comment]
Foi uma atualização do angular, estamos verificando. Basta entrar em contato com a gente que daremos o apoio necessário. Mais uma vez obrigada pela contribuição!
